src/controllers/sendResetToken.js

```diff
@@ -0,0 +1,118 @@
+import bcrypt from 'bcrypt';
+import { randomBytes } from 'crypto';
+import user from '../db/models';
+import sendgrid from '../middlewares/helper';
+
+const { User } = user;
+
+/**
+  * @description class representing reset user password
+  *
+  * @class SendResettoken
+  */
+class SendResetToken {
+  /**
+    * @description - This method is responsible for creating creating password reset token
+    *
+    * @static
+    * @param {object} request - Request sent to the router
+    * @param {object} response - Response sent from the controller
+    *
+    * @returns {object} - object representing response message
+    *
+    * @memberof UserHandler
+    */
+  static async resetToken(request, response) {
+    const { email } = request.body;
+    if (email.trim().length < 1 || !(/\S+@\S+\.\S+/.test(email))) {
+      return response.status(400).json({
+        status: 'Fail',
+        message: 'The email field cannot be empty and it must follow this [example@whatever.com]'
+      })
+    }
+    try {
+      // const {email} = request.body;
+      const userData = await User.findOne({
+        where: { email: request.body.email }
+      });
+      if (!userData) {
+        return response.status(400).json({
+          status: 'Fail',
+          message: 'An email has been sent to your mail',
+          token: request.body.token
+        });
+      }
+      
+      const token = randomBytes(40).toString('hex');
+      const tokenExpiresIn = Date.now() + 3600000; 
+
+      const {username, email, password} = userData.dataValues;
+      const userUpdatedData = await userData.update({
+        token,
+        username,
+        email,
+        password,
+        passwordExpiresIn: tokenExpiresIn || userData.token,
+      });
+      const {HOST_URL} = process.env;
+      const emailResponse = await sendgrid(userData.email, 'noreply@authors.com', HOST_URL, token);
+      response.status(200).json({
+        status: 'Success',
+        message: 'Password reset link has been sent to your mail',
+        token: userUpdatedData.token,
+      });
+    } catch (error) {
+      response.status(500).json({
+        status: 'Error',
+        error: error.message
+      });
+    }
+  }
+
+  /**
+    * @description - This method is responsible for updating user password
+    *
+    * @static
+    * @param {object} request - Request sent to the router
+    * @param {object} response - Response sent from the controller
+    *
+    * @returns {object} - object representing response message
+    *
+    * @memberof UserHandler
+    */
+  static async updateUserPassword(request, response) {
+    const {password, confirmNewPassword, token} = request.body;
+    try {
+      const userToUpdate = await User.findOne({
+        where: { token }
+      });
+      console.log('fnbgiuengwnfwUPSDVSFSF',userToUpdate);
```

Comment:
Hi Emeka, I don't think you need the `console.log` on line 89

Reply:
Thanks for noticing that, I will remove straight away.